packages/labs/router/src/router.ts

```diff
@@ -4,4 +4,82 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-export {};
+import {ReactiveControllerHost} from 'lit';
```

Comment:
```suggestion
import type {ReactiveControllerHost} from 'lit';
```

Reply:
Should we get into the habit of declaring our type-only imports?

Reply:
`typescript-eslint` has support for this (including auto-fix): 

```javascript
    '@typescript-eslint/consistent-type-imports': [
      'error',
      {
        prefer: 'type-imports'
      }
    ],
```